### PR TITLE
Change location of isntackenv.json

### PR DESCRIPTION
**Pull request by smalleni**
It is better to have this location local to the jetpack directory instead of creating
it in the user home directory.

### CODE REVIEW

**Comment by zulcss** [comment]
yes please

**Comment by masco** [comment]
Can one of the admins verify this patch?

File: group_vars/all.yml [hunk 1]
```diff
@@ -2,9 +2,9 @@
 infrared_dir: "{{ ansible_user_dir }}/infrared"
 infrared_workspaces_dir: "{{ ansible_user_dir }}/.infrared/.workspaces"
 infrared_hosts_file: "{{ infrared_workspaces_dir }}/active/hosts"
-instackenv_file: "{{ ansible_user_dir }}/instackenv.json"
+instackenv_file: "{{ playbook_dir }}/instackenv.json"
```
Comment (masco) [inline group_vars/all.yml:5]: where the playbook_dir defined?

Comment (smalleni) [reply group_vars/all.yml:5]: @masco It's an ansible special var.
playbook_dir contains the playbook base directory.

Comment (masco) [reply group_vars/all.yml:5]: ack

**Comment by masco** [comment]
we need to add this line in ci script ```cp ~/instackenv.json .``` and modify the variables in ci/all_osp13.yml

**Comment by smalleni** [comment]
@masco Why didn't the CI run on this?

**Comment by masco** [comment]
run tests

**Pull request merged**